src/partials/header.html

```diff
@@ -1,14 +1,107 @@
 <header class="header">
   <div class="container">
+    <a href="../pages/home.html" class="logo">
+      <svg class="icon icon-logo" width="132" height="24">
+        <use href="../img/icons-header.svg#icon-logo"></use>
+      </svg>
+    </a>
+    <button class="burger-menu">
+      <svg class="icon icon-burger" width="32" height="32">
+        <use href="../img/icons-header.svg#icon-burger"></use>
+      </svg>
+    </button>
     <nav class="nav">
       <ul class="navbar-list">
         <li class="navbar-item">
-          <a href="/" class="router-link">Home</a>
+          <a href="../pages/home.html" class="router-link">Home</a>
         </li>
         <li class="navbar-item">
-          <a href="/favorites" class="router-link">Favorites</a>
+          <a href="../pages/favorites.html" class="router-link">Favorites</a>
         </li>
       </ul>
     </nav>
+    <ul class="social-list">
+      <li class="social-item">
+        <a
+          href="https://www.facebook.com/goITclub/"
+          target="_blank"
+          class="social-link"
+        >
+          <svg class="icon icon-facebook" width="28" height="28">
+            <use href="../img/icons-header.svg#icon-facebook"></use>
+          </svg>
+        </a>
+      </li>
+      <li class="social-item">
+        <a
+          href="https://www.instagram.com/goitclub/"
+          target="_blank"
+          class="social-link"
+        >
+          <svg class="icon icon-instagram" width="28" height="28">
+            <use href="../img/icons-header.svg#icon-instagram"></use>
+          </svg>
+        </a>
+      </li>
+      <li class="social-item">
+        <a
+          href="https://www.youtube.com/c/GoIT"
+          target="_blank"
+          class="social-link"
+        >
+          <svg class="icon icon-youtube" width="28" height="28">
+            <use href="../img/icons-header.svg#icon-youtube"></use>
+          </svg>
+        </a>
+      </li>
+    </ul>
+  </div>
+  <div class="burger-menu-content">
+    <button class="close-menu">
+      <svg class="icon icon-close" width="32" height="32">
+        <use href="../img/icons-header.svg#icon-close"></use>
+      </svg>
+    </button>
+    <nav class="burger-nav">
+      <ul>
+        <li><a href="../pages/home.html">Home</a></li>
+        <li><a href="../pages/favorites.html">Favorites</a></li>
```

Comment:
link should be to the page `'/'` and `'/favorites'`
Please add class `.router-link`. This class is used only for router, not for styling